### PR TITLE
fix: standardize svg formatting

### PR DESCRIPTION
**Pull request by braddialpad**
## Description
Some devs are having issues with certain system svgs not changing color with the font as expected, trying to standardize our svg format a bit to mitigate this. Changed all instances of the black fills on our system svgs to "currentColor" as that makes the most sense.

Also fixed the barge icon which had a white path overtop of a rect so it wouldn't change with the background. Converted this to a path with `fill-rule="evenodd"` to "cut a hole" in the rectangle instead. @francisrupert this should maybe be updated in figma as well.

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [ ] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

### CODE REVIEW

**Review by francisrupert — COMMENTED** [review]
Hmm, I thought this was covered well enough by https://github.com/dialpad/dialtone/blob/staging/gulpfile.js#L271-L307

**Comment by braddialpad** [comment]
Must have been some anomalies not included in there. I'll see if I can update that file to cover more things instead of changing the svgs

**Comment by braddialpad** [comment]
Updated gulpfile to handle the cases we were missing.

File: gulpfile.js [hunk 1]
```diff
@@ -272,6 +272,8 @@ const buildSystemSVGs = function (done) {
     .pipe(cache('buildSystemSVGs'))
     .pipe(replace(' fill="none"', ''))
     .pipe(replace(' fill="#000"', ''))
+    .pipe(replace(' fill="#000000"', ''))
+    .pipe(replace(' fill="#0D0C0F"', ''))
```
Comment (francisrupert) [inline gulpfile.js:276]: A bit whack-a-mole. But yeah, I suppose the `#000000` might've been the real culprit and `#000` naturally wouldn't match those.

**Review by francisrupert — APPROVED** [review]

**Pull request merged**